jclklib/client/notification_msg.cpp

```diff
@@ -109,10 +113,39 @@ PROCESS_MESSAGE_TYPE(ClientNotificationMessage::processMessage)
 		client_ptp_data.gmPresent_event_count.fetch_add(1, std::memory_order_relaxed);
 	}
 
+	if (composite_eventSub[0]) {
+		old_composite_event = composite_client_ptp_data.composite_event;
```

Comment:
Just a note. maybe if this composite event test is reused again elsewhere, we can consider to make a small helper function.

Reply:
yes, you are right. we plan to refactor parse_buffer and process_message function of subscribe and notify message in next PR.